### PR TITLE
Support for non-Swiss projected WMS sources

### PR DESCRIPTION
**Pull request by mariavkourouni**
[Demo](https://mf-geoadmin3.dev.bgdi.ch/kom_wms/)

### CODE REVIEW

**Comment by mariavkourouni** [comment]
Compare 'http://owsproxy.lgl-bw.de/owsproxy/ows/WMS_Maps4BW' (which contains the swiss projection) to 'http://www.webatlasde.de/arcgis/services/Maps4BW/MapServer/WMSServer' (which does not contain swiss projection).

**Comment by oterral** [comment]
I think there is a problem the WMS 'http://www.webatlasde.....'   doesn't contains the swiss projection in its GetCapabilities but the request is done with it http://www.webatlasde.de/arcgis/services/Maps4BW/MapServer/WMSServer?SERVICE=WMS&VERSION=1.3.0&REQUEST=GetMap&FORMAT=image%2Fpng&TRANSPARENT=true&LAYERS=Halbtonvariante&CRS=EPSG%3A21781&STYLES=&WIDTH=1920&HEIGHT=502&BBOX=638094.4%2C266792.60000000003%2C638286.4%2C266842.8

File: src/components/importwms/ImportWmsDirective.js [hunk 1]
```diff
@@ -154,13 +154,6 @@ goog.require('ga_urlutils_service');
                                     l.Abstract) || '';
           };
 
-          // Test if the layer can be displayed with a specific projection
-          var canUseProj = function(layer, projCode) {
-            var projCodeList = layer.CRS || layer.SRS || [];
-            return (projCodeList.indexOf(projCode.toUpperCase()) != -1 ||
-                projCodeList.indexOf(projCode.toLowerCase()) != -1);
-          };
```
Comment (oterral) [inline src/components/importwms/ImportWmsDirective.js]: Keep this function you need it.

**Comment by oterral** [comment]
See  #2917

**Pull request closed without merging**